ord_schema/message_helpers_test.py

```diff
@@ -380,8 +380,8 @@ def test_set_solute_moles(self):
         message_helpers.set_solute_moles(solute, [solvent3],
                                          '3 mM',
                                          overwrite=True)
-        self.assertEqual(solute.amount.moles,
-                         reaction_pb2.Moles(units='NANOMOLE', value=225))
+        self.assertEqual(solute.amount.moles.units, reaction_pb2.Moles.NANOMOLE)
```

Comment:
why did you change this test? proto equality seems better than field equality

Reply:
I forgot why, too... rerunning the old code shows this error. It seems to be due to a subtle reording of floating point ops / units rescaling of nanomoles and whatnot. 

Traceback (most recent call last):
  File "./ord_schema/message_helpers_test.py", line 386, in test_set_solute_moles
    reaction_pb2.Moles(units='NANOMOLE', value=225))
AssertionError: value: 225.00001525878906
units: NANOMOLE
 != value: 225.0
units: NANOMOLE